testing/e2e/Makefile

```diff
@@ -40,9 +39,11 @@ download-agent: set-arch
 
 .PHONY: create-cluster
 create-cluster:
-	@$(call check_defined, SLACK_CHANNEL, SLACK_CHANNEL is not defined. Specify a slack channel to send notifications to.)
 	@$(call check_defined, GITHUB_TOKEN, GITHUB_TOKEN is not defined. Specify a github token to use for the cluster.)
-	oblt-cli cluster create custom --template-file $(CURDIR)/$(TEMPLATE_FILE) --output-file $(CURDIR)/$(CLUSTER_INFO_FILE) --parameters '{"StackVersion": "$(STACK_VERSION)", "ClusterName": "$(CLUSTER_NAME)", "SlackChannel": "$(SLACK_CHANNEL)"}' --wait 15
```

Comment:
We remove the `SlackChannel` and `ClusterName` because are automatically passed by `oblt-cli`. The `ClusterName` is autogenerated and the `SlackChannel` is in the `oblt-cli` configuration.